orion/tests/test_formatters.py

```
@@ -254,6 +257,264 @@ def test_format_average_produces_xml_element(self):
+        periodic = _make_analysis_result()
+        pull_a = _make_pull_analysis([15.0, 25.0])
+        pull_b = _make_pull_analysis([12.0, 22.0])
+        pulls = [(1111, pull_a), (2222, pull_b)]
+
+        formatter = JUnitFormatter()
+        save_path = str(tmp_path / "output.xml")
+        formatter.print_and_save_pr(periodic, pulls, save_path)
+
+        with open(str(tmp_path / "output_test-workload.xml"),
+                  encoding="utf-8") as f:
+            tree = ET.parse(f)
```

Ruff (0.15.14)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)